src/api/projects.js

```diff
@@ -0,0 +1,61 @@
+const express = require('express');
+const projetsExample = require('./dev/projectsExamples.json');
+
+const router = express.Router();
+
+router.get('/', (req, res) => {
+  res.status(200).json(projetsExample);
+});
+
+router.get('/:id', (req, res) => {
+  const projectID = +req.params.id;
+
+  const result = projetsExample.find((project) => project.id === projectID);
+
+  if (result) {
+    res.status(200).json(result);
+  } else {
+    res.status(404).json({ message: 'project not found' });
+  }
+});
+
+router.post('/', (req, res) => {
+  const init = projetsExample;
+  const newProjects = {
+    id: [...init].pop().id + 1,
+    ...req.body,
+  };
+  init.push(newProjects);
+  res.status(201).json(init[init.length - 1]);
+});
+
+router.put('/:id', (req, res) => {
+  const id = +req.params.id;
+  const index = projetsExample.indexOf(projetsExample.find((projet) => projet.id === id));
+
+  if (index >= 0) {
+    let newProject = projetsExample.find((project) => project.id === id);
+    newProject = {
+      id,
+      ...req.body,
```

Comment:
If the body contain only modified fields, original datas will be deleted (have to do like companies path)